File: C#/Klas.cs
namespace CSharp;

public class Klas 
{
    private List<Student> Students;

    public Klas(List<Student> students)
    {
        Students = new();
    }

    public void AddStudent(Student student)
    {
        Students.Add(student);
    }

    public override string ToString()
    {
        return $"Klas: {string.Join(", ", Students)}";
    }
}

File: Java/src/Klas.java
import java.util.ArrayList;
import java.util.List;

public class Klas 
{
    private List<Student> Students;

    public Klas() 
    {
        Students = new ArrayList<Student>();
    }

    public void AddStudent(Student student) 
    {
        Students.add(student);
    }

    public String toString() 
    {
        return "Klas: " + Students + "";
    }
}
